web/cobrands/fixmystreet-uk-councils/assets.js

```diff
@@ -708,6 +707,11 @@ fixmystreet.assets.centralbedfordshire.not_found = function(layer) {
     }
 };
 
+fixmystreet.assets.centralbedfordshire.trees_relevant = function(options) {
+    return options.category === "Communal Trees" ||
```

Comment:
I think you could have asset_group: ['Trees', 'Communal Trees'] in the config - as that checks both group and caetgory, and not need this function, but perhaps having the function is clearer...

Reply:
Went for dropping the function in favour of the config approach